packages/purpose-process/test/createPurpose.test.ts

```diff
@@ -605,4 +604,73 @@ describe("createPurpose", () => {
       })
     ).rejects.toThrowError(duplicatedPurposeTitle(purposeSeed.title));
   });
+  it("should throw organizationIsNotTheDelegatedConsumer when the requester is the Consumer but there is a Consumer Delegation", async () => {
```

Comment:
This test shall be removed. The Consumer must be allowed to create a delegation even if there is a Consumer delegation because both the Consumer and the Delegate can create purposes. This test shall fail - I think it's passing because the `authData` is for a random tenant, not the Consumer.